vaadin-login-flow-parent/vaadin-login-flow-integration-tests/src/main/java/com/vaadin/flow/component/login/examples/OverlayClassNameView.java

```diff
@@ -0,0 +1,35 @@
+package com.vaadin.flow.component.login.examples;
+
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.component.login.LoginOverlay;
+import com.vaadin.flow.router.Route;
+
+@Route(value = "vaadin-login/overlay-class-name")
+public class OverlayClassNameView extends AbstractView {
+
+    private final LoginOverlay login = new LoginOverlay();
+
+    public OverlayClassNameView() {
+        init(login);
+        login.addClassName("custom");
+        login.addLoginListener(e -> login.close());
+
+        NativeButton button = new NativeButton("Open");
```

Comment:
Let's replace `button` with something like `openOverlay` to make it self-explanatory.